tests/test_testing.py

```diff
@@ -0,0 +1,33 @@
+import types
+
```

Comment:
I would prefer renaming this file with `testing.py` and actual `testing.py` with something like `testing_setup.py` if I got it right. It just makes unnecessary confusion.